exporter/googlecloudexporter/README.md

```diff
@@ -6,57 +6,36 @@ This exporter can be used to send metrics, traces, and logs to Google Cloud Moni
 ## Configuration
 | Field               | Default               | Required | Description |
 | ---                 | ---                   | ---      | ---         |
+| `metric`            |                       | `false`  | The [metric](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/googlecloudexporter#configuration-reference) settings of the exporter. |
```

Comment:
These links should be to the version we're supporting so v0.53.0

Reply:
This should be fixed now.